src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -350,10 +353,17 @@ protected WgsMetrics generateWgsMetrics() {
     }
 
     protected long getBasesExcludedBy(final CountingFilter filter) {
+        if(INTERVALS != null){
+            return filter.getFilteredRecords();
+        }
         return filter.getFilteredBases();
```

Comment:
It's very important to copy over the comment from CollectWgsMetricsFromSampledSites because the behavior is different.  You may even want to mention this in the docs for the INTERVALS argument too.